### ISSUE TITLE
Move energy and running costs [discussion with Lluis]

### ISSUE THREAD

**Issue by ElaChe00** [icra/ecam#295]
I think it would make sense to move energy and running costs to "Water Supply" and "Sanitation". 
Often, you look at it as a whole and utilities don't have the information par stage? We should discuss this.

**Comment by ElaChe00** [comment]
Include this point as survey question for the pilot phase.

**Comment by ElaChe00** [comment]
What if, we give users an option? People who can split the costs are very welcome to do so, but for all the others: They should have the possiblity to include their costs even if they don't know where they come from exactly.

So giving the opportunity to insert costs of water supply and costs of sanitation. It can include a remark such as "_If electricity/operational costs cannot be divded into the stages, please insert them for the overal system (Water supply/Sanitation)"_ 
![image](https://user-images.githubusercontent.com/45791540/107660013-16467380-6c88-11eb-8297-a7ec3e8641ca.png)

![image](https://user-images.githubusercontent.com/45791540/107660044-1e9eae80-6c88-11eb-9994-4c5839792d2a.png)

**Comment by holalluis** [comment]
Create new variables for Water supply and Sanitation and create "Yes/No" questions for substages. The variables in water supply and sanitation have an estimation button that sums the values in substages

**Comment by holalluis** [comment]
![image](https://user-images.githubusercontent.com/1082644/108212323-028a8980-712e-11eb-9e8a-12020f319c83.png)

**Comment by ElaChe00** [comment]
Looks great! Can you rephrase the question to:

**"Do you know the utility costs by stage?"**

**Comment by holalluis** [comment]
![image](https://user-images.githubusercontent.com/1082644/108218229-babb3080-7134-11eb-8526-7bda871ec68d.png)
done

**Comment by ElaChe00** [comment]
Thanks! Looking good
